src/OracleService/OracleService/config.json

```diff
@@ -6,6 +6,9 @@
     "AllowedContentTypes": [ "application/json" ],
     "Https": {
       "Timeout": 5000
+    },
+    "NeoFs": {
```

Comment:
I really like it as an configuration of the Native Oracle service.
However, isn't this a specific use of the Oracle Service? I am not fully sure about this design, if this should be a different plugin.

Reply:
NeoFS is an external system and we talk to external systems via Oracle service, this communication path unification is the main reason for it being here. See neo-project/neo#1270 also.

Reply:
I got this @roman-khimov, however, I still think that there may be some space to design it by inheritance of Oracle Service and its respective modification.

Reply:
I am seeing this comment https://github.com/neo-project/neo/issues/1270#issuecomment-558153166 and seeing this template I think it makes sense to be here, since it is a native feature of Neo as well.